.gitignore

```diff
@@ -4,3 +4,5 @@ tests/adapters/*
 !tests/adapters/*.dist
 bin/*
 !bin/configure_test_env.sh
+/.idea
```

Comment:
Could you remove these two lines?
I doubt that open-source libs should take care about particular IDE.